modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/constants/ContentDashboardWebKeys.java

```diff
@@ -30,4 +30,8 @@ public class ContentDashboardWebKeys {
 		CONTENT_DASHBOARD_ADMIN_MANAGEMENT_TOOLBAR_DISPLAY_CONTEXT =
 			"CONTENT_DASHBOARD_ADMIN_MANAGEMENT_TOOLBAR_DISPLAY_CONTEXT";
 
+	public static final String[] DEFAULT_INTERNAL_VOCABULARIES = {
```

Comment:
Hey @Stephy6 !!!

Could you create a new file called ContentDashboardConstants and move DEFAULT_INTERNAL_VOCABULARIES there?

Also, do you mind to replace every use of "audience" and "stage" by this new contants?

Reply:
I guess you need a map, an array would not do the job 🤔

Reply:
Why not? They are constants so I think a dynamic structure isn't needed, and it requieres more memory

Reply:
Should I replace them also in the tests?

Reply:
hmmm... if the constant is an array you'll need to reference the values by it index which can be fragile...
I would go for something like `DEFAULT_INTERNAL_VOCABULARIES.MY_INTERNAL_VOCABULARY`

Reply:
Hey @Stephy6 !!

Sorry, I suppose you have more Java context. We have [Enums](https://www.baeldung.com/a-guide-to-java-enums) in Java, maybe it is the right structure you can use.

But if you want to use 2 constants it is OK too. 

An array should not be used since we only have 2 constants and we don't intent to make any of them dynamic, and the same with the map, we don't need to add more complexity when we don't need it.

Once we make the vocabularies configurable, maybe we can use a more complex structure, but we don't need it right now.

Thanks!!